Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/PropertyEditorNodes.h

```diff
@@ -204,6 +204,11 @@ namespace AZ::DocumentPropertyEditor::Nodes
         static constexpr auto OnActivate = CallbackAttributeDefinition<void()>("OnActivate");
     };
 
+    struct PrefabOverrideProperty : PropertyEditorDefinition
+    {
+        static constexpr AZStd::string_view Name = "PrefabOverrideProperty";
+    };
```

Comment:
Similar to comment above. My understanding is the intention for PropertyEditorNodes and DocumentSchema to contain the general stuff used by many editors, and for more specific impl to go in files focused around that particular area.

That being said, I think this is one of the first implementations specific to a particular system, like prefabs. If it isn't possible yet then it definitely won't break anything to leave it here.

@alexmontAmazon Does that sound right?

Reply:
+1. Confine prefab-related things to a prefab file. We hope to eventually have *tons* of editors based on the DPE, so we can't afford to jam everything into the base.

Reply:
Moved this to AzToolsFramework/Prefab/DocumentPropertyEditor/PrefabPropertyEditorNodes.h